### PR TITLE
update docker image references to live tags

### PR DESCRIPTION
**Pull request by cleary**
Hi @ramonski

refs #16 - the issue was raised because it seem like the default docker-compose.yml should reference a working dockerhub image tag

This patch provides that -

Regards, 
Bernie

### CODE REVIEW

File: 2.3.0/docker-compose.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ services:
       ZEO_ADDRESS: "zeo:8080"
 
   instance2:
-    image: senaite-latest
+    image: senaite/senaite:edge
```
Comment (ramonski) [inline 2.3.0/docker-compose.yml]: Please use the tag `senaite/senaite:2.3.0` here

File: 2.2.0/docker-compose.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ services:
       ZEO_ADDRESS: "zeo:8080"
 
   instance2:
-    image: senaite-latest
+    image: senaite/senaite:edge
```
Comment (ramonski) [inline 2.2.0/docker-compose.yml]: Please use the tag `senaite/senaite:2.2.0` here

**Review by ramonski — CHANGES_REQUESTED** [review]
Hi @cleary,
Thanks for your PR and apologies for the misunderstanding!
Would you mind to change the tags for the tagged folders accordingly?
Thanks!

**Comment by cleary** [comment]
That should be done - thanks :)

**Review by ramonski — APPROVED** [review]
Thanks @cleary!

**Pull request merged**